tests/functional/ContentRepositoryTest.php

```diff
@@ -489,6 +489,76 @@ public function it_checks_if_parent_is_proper_type()
         );
     }
 
+    /**
+     * @test
+     */
+    public function it_should_retrive_id_of_non_trashed_content() {
```

Comment:
This metod returns whole content, not only it's id, so an more accurate name will be "it_should_retrive_non_trashed_content", same for blocks tests.